auth.go

```diff
@@ -50,12 +53,16 @@ func newAuthAddr(addr net.Addr) string {
 // Auth stores lookups for bans, whitelists, and ops. It implements the sshd.Auth interface.
 // If the contained passphrase is not empty, it complements a whitelist.
 type Auth struct {
-	passphraseHash []byte
-	bannedAddr     *set.Set
-	bannedClient   *set.Set
-	banned         *set.Set
-	whitelist      *set.Set
-	ops            *set.Set
+	passphraseHash  []byte
+	whitelistModeMu sync.RWMutex
+	whitelistMode   bool
+	bannedAddr      *set.Set
+	bannedClient    *set.Set
+	banned          *set.Set
+	whitelist       *set.Set
+	ops             *set.Set
+	opFile          string
```

Comment:
What do you think about changing this to something like `opsLoader func() ([]ssh.PublicKey, error)`? Then we can specify the loader in `package main` so this package doesn't need to know anything about the filesystem etc.

(I prefer to keep as much of the state defined in main as possible, makes it easier to test/refactor. This way `fromFile` and friends can be moved back to cmd.)